KenBurns/Classes/KenBurns.swift

```diff
@@ -166,13 +184,14 @@ func ==(lhs: KenBurnsAnimation, rhs: KenBurnsAnimation) -> Bool {
     }
 
     public func setImage(_ image: UIImage) {
+        index = -1
         currentImageView.image = image
         nextImageView.image = image
     }
 
     public func fetchImage(_ url: URL, placeholder: UIImage?) {
         [ currentImageView, nextImageView ].forEach {
-            $0.sd_setImage(with: url, placeholderImage: placeholder)
+            $0.kf.setImage(with: url, placeholder: placeholder, options: [.transition(.fade(0.2))])
```

Comment:
With this change, do we still need a depndency on `SDWebImage` pod at all.

Reply:
👍🏼you're right, removed